eurovision/notebooks/story.ipynb

```diff
@@ -1136,15 +1186,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this plot, countries are grouped by ... **TODO** \n",
+    "In this plot, countries are grouped using hierarchical clustering, ordering countries with similar voting patterns together. The heatmap shows the voting deviations from the mean each country receives. Lighter colors indicate higher deviations from the mean. \n",
     "\n",
-    "There are a number of interesting details in this plot.\n",
-    "For example, the black boxes indicate that Turkey's entries have consistently obtained higher-than-average scores from Germany, the Netherlands, Belgium, and France; but that this isn't reciprocated.\n",
-    "A possible hypothesis for this may be the fact there is a significant Turkish diaspora in these countries: according to Pashayan (2012), there were 10 million \"Euro-Turks\" in these four countries in 2012.\n",
+    "There are a number of interesting details in this plot highlighted by different colored boxes.  \n",
     "\n",
-    "This suggests that migration data between two countries might be another useful thing to look at.\n",
+    "For example, the small blue boxes on the left indicate that Turkey's entries have consistently obtained higher-than-average scores from Germany, the Netherlands, Belgium, and France; but that this isn't reciprocated, as the other blue box on the top shows.\n",
+    "A possible hypothesis for this may be the fact there is a significant Turkish diaspora in these countries: according to Pashayan (2012)*, there were 10 million \"Euro-Turks\" in these four countries in 2012. This suggests that migration data between two countries might be another useful thing to look at.\n",
     "\n",
-    "REF (to be done up properly, and actually just ref 5 from https://en.wikipedia.org/wiki/Turkish_diaspora): Pashayan, Araks (2012), \"Integration of Muslims in Europe and the Gülen\", in Weller, Paul; Ihsan, Yilmaz (eds.), European Muslims, Civility and Public Life: Perspectives On and From the Gülen Movement, Continuum International Publishing Group, ISBN 978-1-4411-0207-2."
+    "The large black boxes show clusters of countries who vote highly for each other and tend to be closer geographically. For example, the cluster in the bottom right contains many skandinavian countries, the box in the middle balkan countries and the top box ex-soviet countries. overall, this suggests that sharing a border and being close geographically matters for how countries vote.\n",
```

Comment:
```suggestion
    "The large black boxes show clusters of countries who vote highly for each other and tend to be closer geographically. For example, the cluster in the bottom right contains many Nordic countries, the box in the middle balkan countries and the top box ex-soviet countries. overall, this suggests that sharing a border and being close geographically matters for how countries vote.\n",
```
I wonder if "Nordic" (Denmark, Finland, Iceland, Norwa, Sweden) might be better than Scandinavian (Denmark, Norway, Sweden) here?